src/device_agent.py

```diff
@@ -709,7 +709,17 @@ def get_sensor():
         parts = output.split(" ")
         # Extracted video device file name copied to dev_num variable
         dev_num = parts[6]
-        dev_no = dev_num.replace("/dev/", "")
+        # Extract actual device id
+        dev_no = subprocess.Popen(
+            "readlink -f {}".format(dev_num),
+            stdout=subprocess.PIPE,
+            bufsize=1,
+            universal_newlines=True,
+            shell=True,
+        )
```

Comment:
Use some python function to get the soft link
There should be something from import os

Reply:
Done.